src/testnet/helium/run_loop.rs

```diff
@@ -15,7 +15,7 @@ pub struct RunLoop {
     config: Config,
     pub node: Node,
     new_burnchain_state_callback: Option<fn(u64, &BurnchainState)>,
-    new_tenure_callback: Option<fn(u64, &LeaderTenure)>,
+    new_tenure_callback: Option<fn(u64, &mut LeaderTenure)>,
```

Comment:
Why is this callback providing a mutable reference?

Reply:
The internal `MemPoolDB` needs to be mutable in order to store transactions.